test/models/parted_test.rb

```diff
@@ -12,7 +12,7 @@ class PartedTest < ActiveSupport::TestCase
     assert_not edition.valid?
 
     assert_equal({ title: ["can't be blank"] }, edition.errors[:parts][0]["54c10d4d759b743528000010:1"])
-    assert_equal({ slug: ["can't be blank", "is invalid"] }, edition.errors[:parts][0]["54c10d4d759b743528000011:2"])
+    assert_equal({ slug: ["can't be blank"] }, edition.errors[:parts][0]["54c10d4d759b743528000011:2"])
```

Comment:
It seems strange that this error is gone (to support first comment)

Reply:
See https://github.com/alphagov/publisher/pull/1486#discussion_r688477271